src/extension.ts

```diff
@@ -1812,8 +1812,13 @@ async function deleteContextKubernetes(explorerNode: explorer.KubernetesObject)
     }
 }
 
-function copyKubernetes(explorerNode: explorer.KubernetesObject) {
-    clipboard.writeSync(explorerNode.id);
+async function copyKubernetes(explorerNode: explorer.KubernetesObject) {
+    const clipboard = (<any>vscode.env).clipboard;
+    if (clipboard) {
+        await clipboard.writeText(explorerNode.id);
+    } else {
+        await clipboard.write(explorerNode.id);
```

Comment:
Local `clipboard` will shadow the imported `clipboard` won't it?  In VS Code builds without `env.clipboard` I think this might fail with an undefined error; at best, the different meanings of the name are confusing.  Change the name of the local or the import.

Reply:
fixed.